You're not authorized to run analysis. Please contact the project administrator

Latest 9.9 sonarqube image gives error. 

```
> Task :sonarqube FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sonarqube'.
> You're not authorized to run analysis. Please contact the project administrator.

```

The only possible solution I found to force it work out of the box is to use deprecated properties:
sonar.login=admin
sonar.password=admin

--- a/build.gradle
+++ b/build.gradle
@@ -1,7 +1,7 @@
 plugins {
     id 'java'
     id 'jacoco'
-    id 'org.sonarqube' version '3.3'
+    id 'org.sonarqube' version '4.0.0.2929'
 }
 
 repositories {
@@ -40,6 +40,8 @@ jacocoTestReport {
 sonarqube {
     properties {
         property 'sonar.host.url', 'http://localhost:9000'
+        property 'sonar.login', 'admin'
+        property 'sonar.password', 'admin'
     }
 }
 

